action.yml

```
@@ -54,3 +54,7 @@ inputs:
     description: 'Email address used to configure git (for git operations)'
     required: false
     default: 'github-actions[bot]@users.noreply.github.com'
+  commitizen_version:
+    description: 'Specify the version to be used by commitizen'
+    required: false
+    default: latest
```

And I'm not sure it's an issue, but to be consistent with other defaults maybe this should be quoted

```suggestion
    default: 'latest'
```